shell/probe/raindrop.ps1

```diff
@@ -1,46 +1,39 @@
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
 function Run {
     Param([String]$Dat = "")
 
-    $TempFile = [System.IO.Path]::GetTempFileName() | Rename-Item -NewName { [System.IO.Path]::ChangeExtension($_, "exe") } -PassThru
-
+    $Vst = New-Item -Path "$Dir\$(New-Guid).exe" -Force
     $Headers = @{
-        'token' = $Token
+        'token' = FromEnv "PRELUDE_TOKEN"
```

Comment:
Since we read the token every time, if the token changes in the env different runs will beacon with different tokens

Reply:
yep; we do the same thing in nocturnal (by design) -- so we can modify the value on the fly, if needed